src/pages/tBTC/Bridge/components/MintingTransactionDetails.tsx

```diff
@@ -31,10 +48,31 @@ const MintingTransactionDetails = () => {
         precision={6}
         higherPrecision={8}
       />
-      <TransactionDetailsItem
-        label="ETH address"
-        value={shortenAddress(ethAddress)}
-      />
+      {withTotal || withSubTotal ? (
+        <ListItem>
+          <StackDivider w="full" h="1px" bg="hsla(0, 0%, 100%, 10%)" />
+        </ListItem>
+      ) : null}
+      {withSubTotal ? (
+        <TransactionDetailsAmountItem
+          label="Total Fees"
+          tokenAmount={totalFees.toString()}
+          tokenSymbol="tBTC"
+          precision={6}
+          higherPrecision={8}
+          isSubTotal={withSubTotal}
+        />
+      ) : null}
+      {withTotal ? (
+        <TransactionDetailsAmountItem
+          label="You will receive"
+          tokenAmount={amountToReceive.toString()}
+          tokenSymbol="tBTC"
+          precision={6}
+          higherPrecision={8}
+          isTotal={withTotal}
+        />
+      ) : null}
```

Comment:
To be honest I'm little bit confused why `withSubTotal` is related to **Total** fees, and why `withTotal` is related to `receivedAmount` 🤔 In this case, maybe we should rename those props to `withTotalFees` and `withReceiveAmount` respectively.